transaction/src/transaction.rs

```diff
@@ -78,6 +78,7 @@ impl Transaction {
         })
     }
 
+    // TODO: make sure payloads from Swap actions included
```

Comment:
just checking, this is blocked on including the actions in the `Transaction` enum right?

Reply:
Yes